pyvista/plotting/mapper.py

```diff
@@ -322,9 +346,10 @@ def set_custom_opacity(
             rgba[:, :-1] = Color(color, default_color=theme.color).int_rgb
             rgba[:, -1] = np.around(opacity * 255)
 
-            self.configure_scalars_mode(
+            new_scalars = self.configure_scalars_mode(
```

Comment:
In light of the tuple, the name `new_scalars` here and in `plotting.py` might be misleading.

Reply:
Resolved in 32fcae6f21b8213e6c988a2d78a960e9a0300d09.